packages/nocodb/src/lib/version-upgrader/ncStickyColumnUpgrader.ts

```diff
@@ -38,6 +38,50 @@ export default async function ({ ncMeta }: NcUpgraderCtx) {
       view_columns_meta.push(col_meta);
     }
 
+    // if no display value column is set
+    if (!view_columns_meta.some((column) => column.pv)) {
+      const pkIndex = view_columns_meta.findIndex((column) => column.pk)
+  
+      // if PK is at the end of table
+      if (pkIndex === view_columns_meta.length - 1) {
+        if (pkIndex > 0) {
+          await ncMeta.metaUpdate(
+            null,
+            null,
+            MetaTable.COLUMNS,
+            { pv: true },
+            view_columns_meta[pkIndex - 1].id
+          );
+        } else if (view_columns_meta.length > 0) {
+          await ncMeta.metaUpdate(
+            null,
+            null,
+            MetaTable.COLUMNS,
```

Comment:
If I'm not wrong it needs to be in a new upgrader file otherwise, it won't run on the user's machine who had already started using the latest version.

Reply:
agree

Reply:
I proposed changing upgrader version to latest instead of creating a new upgrade file to keep code clean as this upgrader doesn't have any side effects, waiting for pranav to confirm if it is appropriate.

Reply:
@mertmit : that sounds good, we can do it in that way :+1: